### ISSUE TITLE
Bootstrap theme | If a photo needs to be approved 1 time on 40-50 gets glitched

### ISSUE THREAD

**Issue by optiroot** [moxi9/phpfox#803]
If you set that an user group need that a photo must be approved before it gets displayed publicly, sometimes could happen that the moderator is going to approve/delete the photo doesn't see the photo itself. That's just a glitch because the photo seems to be "empty" and it goes above the page.
I have a screenshot made by my phone (it's rare to have this glitch shown) but is the only that I got; sorry for the bad quality:
![photo1](https://cloud.githubusercontent.com/assets/12573563/11710242/fc63083a-9f1d-11e5-9048-481a1545efa5.jpg)

As you can see the image is above the page practically, and if you open it the loading circle keeps going infinite.
The image must be deleted and re-uploaded to be shown. If you approve the image, sometimes it's shown and sometimes not.

I know that this bug is difficult to reproduce due to its nature but I believe that with the screenshot could be a little easier.

Thank you!

**Comment by Data66PF** [comment]
Tested with bootstrap and neutron themes as this doesn't say which. Using iPhone 5c with default safari browser. Site is running v4.1.1 build 3. I can't reproduce this yet.

**Comment by Data66PF** [comment]
Can you still reproduce this in 4.1.2?

**Comment by optiroot** [comment]
Actually it isn't happened still!
But if it happens my staff will let me know and I'll publish here!

**Comment by neil96** [comment]
Please open new issue when it happens again.
